rust/operator-binary/src/config.rs

```diff
@@ -92,15 +92,25 @@ pub fn build_bootstrap_conf(
     // Configure where NiFi's lib and conf directories live
     bootstrap.insert("lib.dir".to_string(), "./lib".to_string());
     bootstrap.insert("conf.dir".to_string(), "./conf".to_string());
-    // How long to wait after telling NiFi to shutdown before explicitly killing the Process
-    bootstrap.insert("graceful.shutdown.seconds".to_string(), "20".to_string());
+    // TODO: How long to wait after telling NiFi to shutdown before explicitly killing the Process
```

Comment:
Please extract this into a function in graceful_shutdown.rs similar to https://github.com/stackabletech/kafka-operator/blob/b6aa6613989660932b0b18af68404914e2153c25/rust/operator/src/operations/graceful_shutdown.rs#L15
Also, you can simply set the property if `nifi_config.graceful_shutdown_timeout` is Some() and do nothing otherwise, as the merge mechanism will always provide a value.

Reply:
https://github.com/stackabletech/nifi-operator/pull/528/commits/4c4ab90c16f24f9a4928b29604321eccb7ed2b52